### PR TITLE
Beta

### PR DESCRIPTION
**Pull request by dbiegan**
The beta branch contains most recent work from dev plus features implemented from my old team. Conflicts can be resolved by selecting beta changes. Thank you!

### CODE REVIEW

**Comment by dbiegan** [comment]
Nevermind, just one conflict in pom.xml. Now resolved!

**Review by Rephaelbj — CHANGES_REQUESTED** [review]
This PR has some changes in it that were merged already in a different way, i.e. the edit team velocity.

**Review by dbiegan — COMMENTED** [review]
Changes made!

File: ScrumBoardSimulator/src/main/java/se/bettercode/scrum/ScrumGameApplication.java [hunk 1]
```diff
@@ -300,4 +348,204 @@ public void stop() {
     private BurnupChart getNewBurnupChart() {
         return new BurnupChart(SPRINT_LENGTH_IN_DAYS, false);
     }
+    
+    /* This method is used to build the Stage for editing the team attributes. */
+    private void buildEditTeamStage() {
+    	Stage editTeamStage = new Stage();
```
Comment (Rephaelbj) [inline ScrumBoardSimulator/src/main/java/se/bettercode/scrum/ScrumGameApplication.java:354]: Deleted this

File: ScrumBoardSimulator/src/main/java/se/bettercode/scrum/ScrumGameApplication.java [hunk 2]
```diff
@@ -300,4 +348,204 @@ public void stop() {
...
+    	
+    	editTeamStage.setScene(new Scene(editTeamPane, 350.0, 300.0));
+    	editTeamStage.show();
+    }
+    
+    public void buildBurndownChartStage() {
+
+        Stage burndownChartStage = new Stage();
+        burndownChartStage.setTitle("Burndown Chart");
+
```
Comment (Rephaelbj) [inline ScrumBoardSimulator/src/main/java/se/bettercode/scrum/ScrumGameApplication.java:402]: Please remove

File: ScrumBoardSimulator/src/main/java/se/bettercode/scrum/ScrumGameApplication.java [hunk 3]
```diff
@@ -300,4 +348,204 @@ public void stop() {
...
+        borderpane.setCenter(burnDownChart);
+
+
+        //burndownChartStage.setScene(scene);
+        burndownChartStage.show();
+
+    }
+    
+    /* This method is used to build the Stage for editing the team attributes. */
+    private void buildTaigaIntegrationStage() {
```
Comment (Rephaelbj) [inline ScrumBoardSimulator/src/main/java/se/bettercode/scrum/ScrumGameApplication.java:427]: Please move this to a separate file for modularity. Additionally this makes the file super long, as well as it's different then the rest of the code that does a similar thing. Also there's an option for import from taiga in the file menu. I believe the pop out should be linked to that.

File: ScrumBoardSimulator/src/main/java/se/bettercode/scrum/gui/ToolBar.java [hunk 4]
```diff
@@ -10,10 +10,16 @@
 import javafx.scene.control.ChoiceBox;
 import javafx.scene.control.Tooltip;
 import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
 
 public class ToolBar extends HBox {
 
     private final Button startButton = new Button("Start Sprint");
+    private final Button editButton = new Button("Edit Team");
```
Comment (Rephaelbj) [inline ScrumBoardSimulator/src/main/java/se/bettercode/scrum/gui/ToolBar.java:18]: Remove

**Review by Rephaelbj — CHANGES_REQUESTED** [review]

**Review by bkanumuri1 — APPROVED** [review]

**Review by Rephaelbj — APPROVED** [review]

**Pull request merged**